client/scripts/DrawTogether.js

```diff
@@ -1632,11 +1760,87 @@ DrawTogether.prototype.resetProtectedRegions = function () {
 			this.chat.addMessage("Regions", "Reset Error: " + err);
 			return;
 		}
-
+		this.getMyProtectedRegions();
 		this.chat.addMessage("Regions", "Reset your regions");
 	}.bind(this));
 };
 
+DrawTogether.prototype.removeProtectedRegion = function (regionId, element) {
+	this.network.socket.emit("removeprotectedregion", regionId, function (err, result) {
+		if (err) {
+			this.chat.addMessage("Regions", "Reset Error: " + err);
+			return;
+		}
+		element.remove();
+		this.getMyProtectedRegions();
+		this.chat.addMessage("Regions", "Removed the region");
+	}.bind(this));
+};
+
+DrawTogether.prototype.getMyProtectedRegions = function (callback) {
+	this.network.socket.emit("getmyprotectedregions", function (err, result) {
+		if (err) {
+			this.chat.addMessage("Getting Protected Regions", "Error: " + err);
+			return;
+		}
+		this.regionsList = JSON.parse(result);
+
+		if(typeof callback == "function")
+			callback();
+	}.bind(this));
+};
+
+DrawTogether.prototype.addUsersToMyProtectedRegion = function (userIdArr, regionId, callback) {
+	console.log("userIdArr", userIdArr);
+	this.network.socket.emit("adduserstomyprotectedregion", userIdArr, regionId, function (err, result) {
+		if (err) {
+			this.chat.addMessage("Adding users to protected region", "Error: " + err);
+			return;
+		}
+		setTimeout(function(){
+			this.getMyProtectedRegions(callback);
+		}.bind(this), 1000);
+		
+		this.chat.addMessage("Regions", result.success);
+		console.log("result:",result);
+		//this.regionsList = JSON.parse(result);
+	}.bind(this));
+};
+
+DrawTogether.prototype.removeUsersToMyProtectedRegion = function (userIdArr, regionId, callback) {
+	console.log("userIdArr", userIdArr);
+	this.network.socket.emit("removeuserstomyprotectedregion", userIdArr, regionId, function (err, result) {
```

Comment:
In english it would be remove users FROM my protected region so even if this might sound more consistent I think keeping it readable might be better.

These are also very long so they are a bit hard to read. Maybe somehting like "protectedregion.removeuser" ? 
Or maybe do it camelCase?